packages/lib-core/src/store/PluginLoader.ts

```diff
@@ -150,6 +158,24 @@ export class PluginLoader {
 
     this.plugins.set(pluginName, data);
 
+    /**
+     * Inject runtime script if it exists in a separate chunk
+     */
+    if (manifest.runtimeChunkScript) {
```

Comment:
Loading a separate runtime chunk (if present) should happen right before loading the actual plugin entry script, i.e. after resolving the plugin's dependencies.

This can be done as a follow-up improvement.